src/app/body/board/performance-column-chart/performance-column-chart.component.ts

```diff
@@ -10,7 +11,7 @@ import { BackendService } from 'src/app/services/backend/backend.service';
 })
 export class PerformanceColumnChartComponent implements OnInit {
 
-  constructor(public db: AngularFirestore, public backendService: BackendService) { }
+  constructor(public db: AngularFirestore, public backendService: BackendService, private functions: AngularFireFunctions) { }
```

Comment:
Now that we are not using db here and rather we are doing everything in api. let's remove this.